plugins/katello/3.18/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,10 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important notes:
+
+* Pulpcore installs a separate PostgreSQL database. Databases should be upgraded to avoid conflicts.
+
 In either scenario, both of the databases don't have to be remote. You can opt to use only a remote mongo database or only a remote postgresql database. Both postgresql and mongo databases can be on the same host, but this isn't recommended due to the amount of resources mongo can use.
```

Comment:
I think we need the "Pulpcore database related:" section below here for 3.18.

After that hopefully we can merge?

Reply:
@ianballou I will get to adding that asap.

Reply:
Cool. We'll need someone with merge access to give it a final look over. Maybe @ekohl since he was already in here.